other/install_2-1_mysql.sql

```diff
@@ -2145,13 +2145,15 @@ VALUES ('smfVersion', '{$smf_version}'),
 	('defaultMaxListItems', '15'),
 	('loginHistoryDays', '30'),
 	('httponlyCookies', '1'),
+	('samesiteCookies', 'lax'),
 	('tfa_mode', '1'),
 	('export_dir', '{$boarddir}/exports'),
 	('export_expiry', '7'),
 	('export_min_diskspace_pct', '5'),
 	('export_rate', '250'),
 	('allow_expire_redirect', '1'),
 	('json_done', '1'),
+	('attachments_21_done', '1'),
```

Comment:
Not apart of this PR?

Reply:
Yes, as noted in the initial post, I caught this minor omission when looking at the install scripts.